single_tech_samples/storage/sample1_data_lifecycle/terraform/modules/services/datalifecycle/main.tf

```diff
@@ -0,0 +1,144 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "3.22.0"
+    }
+    azurecaf = {
+      source = "aztfmod/azurecaf"
+      version = "1.2.11"
+    }
+  }
```

Comment:
Enforce required terraform version here. For reference, please check https://github.com/Azure-Samples/virtual-network-integration-recipes/blob/d4e65885892241fc95c164275738cdacf4130dc1/src/az-func-http/deploy/terraform/main.tf#L18

Reply:
Done